### ISSUE TITLE
Sync issue - remove not possible

### ISSUE THREAD

**Issue by HyperCriSiS** [syncthing/syncthing#1609]
Hi,

whats the issue there?
I deleted the files in this folder and copied other files into it.

Greetings
Hyper

[HM5EV] 19:34:47 INFO: Puller (folder "Appz", dir "Archiv\\Uniextract\\Bin"): delete: remove D:\CloudStorage\Appz\Archiv\Uniextract\Bin: Das Verzeichnis ist nicht leer.
[HM5EV] 19:34:47 INFO: Puller (folder "Appz", dir "Archiv\\Uniextract\\Bin"): delete: remove D:\CloudStorage\Appz\Archiv\Uniextract\Bin: Das Verzeichnis ist nicht leer.
[HM5EV] 19:34:47 INFO: Puller (folder "Appz", dir "Archiv\\Uniextract\\Bin"): delete: remove D:\CloudStorage\Appz\Archiv\Uniextract\Bin: Das Verzeichnis ist nicht leer.
[HM5EV] 19:34:47 INFO: Puller (folder "Appz", dir "Archiv\\Uniextract\\Bin"): delete: remove D:\CloudStorage\Appz\Archiv\Uniextract\Bin: Das Verzeichnis ist nicht leer.
[HM5EV] 19:34:48 INFO: Puller (folder "Appz", dir "Archiv\\Uniextract\\Bin"): delete: remove D:\CloudStorage\Appz\Archiv\Uniextract\Bin: Das Verzeichnis ist nicht leer.
[HM5EV] 19:34:48 INFO: Puller (folder "Appz", dir "Archiv\\Uniextract\\Bin"): delete: remove D:\CloudStorage\Appz\Archiv\Uniextract\Bin: Das Verzeichnis ist nicht leer.
[HM5EV] 19:34:48 INFO: Puller (folder "Appz", dir "Archiv\\Uniextract\\Bin"): delete: remove D:\CloudStorage\Appz\Archiv\Uniextract\Bin: Das Verzeichnis ist nicht leer.
[HM5EV] 19:34:48 INFO: Puller (folder "Appz", dir "Archiv\\Uniextract\\Bin"): delete: remove D:\CloudStorage\Appz\Archiv\Uniextract\Bin: Das Verzeichnis ist nicht leer.
[HM5EV] 19:34:48 INFO: Puller (folder "Appz", dir "Archiv\\Uniextract\\Bin"): delete: remove D:\CloudStorage\Appz\Archiv\Uniextract\Bin: Das Verzeichnis ist nicht leer.
[HM5EV] 19:34:49 INFO: Puller (folder "Appz", dir "Archiv\\Uniextract\\Bin"): delete: remove D:\CloudStorage\Appz\Archiv\Uniextract\Bin: Das Verzeichnis ist nicht leer.
[HM5EV] 19:34:49 INFO: Puller (folder "Appz", dir "Archiv\\Uniextract\\Bin"): delete: remove D:\CloudStorage\Appz\Archiv\Uniextract\Bin: Das Verzeichnis ist nicht leer.
[HM5EV] 19:34:49 WARNING: Folder "Appz" isn't making progress - check logs for possible root cause. Pausing puller for 1m0s.

**Comment by AudriusButkevicius** [comment]
Rescanning on the other side should fix this?

**Comment by calmh** [comment]
(Directory not empty)

Or removing any ignored files present in the mentioned Bin dir...

**Comment by HyperCriSiS** [comment]
It happens on both sides :(

**Comment by HyperCriSiS** [comment]
I deleted the folder on both sides and copied them again. Now the error is gone.
Had this on a different folder, which I also deleted.
But now I get this error:
IE3R5] 08:34:34 INFO: Puller (folder "Appz", file "MisC\\Marble\\Data\\settings\\MarblePortableSettings.ini"): dst stat dir: GetFileAttributesEx D:\Appz\MisC\Marble\Data\settings: The System cannot find the specified file..
[IE3R5] 08:34:34 WARNING: Folder "Appz" isn't making progress - check logs for possible root cause. Pausing puller for 1m0s.

**Comment by HyperCriSiS** [comment]
I deleted the folder on both sides and copied them again. Now the error is gone.
Had this on a different folder, which I also deleted.
But now I get this error:

IE3R5] 08:34:34 INFO: Puller (folder "Appz", file "MisC\\Marble\\Data\\settings\\MarblePortableSettings.ini"): dst stat dir: GetFileAttributesEx D:\Appz\MisC\Marble\Data\settings: The System cannot find the specified file..
[IE3R5] 08:34:34 WARNING: Folder "Appz" isn't making progress - check logs for possible root cause. Pausing puller for 1m0s.
